Program.cs

```diff
@@ -1,7 +1,58 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+
+namespace Training {
+   class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter number: ");
+         int num = int.Parse (Console.ReadLine ());

```

Comment:
Negative value crashes the app.
0 shows nothing.
Shouldn't you have asked for positive number?

Reply:
Changed.